### PR TITLE
Create comments.md under postgresql

### PR DESCRIPTION
**Pull request by thiennguyen93**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by thiennguyen93** [comment]
CLA has been signed. However, license/cla is still pending

**Comment by letsbuildanything** [comment]
@thiennguyen93, it appears that you've submitted this PR using two accounts. Therefore, please sign this CLA with both of your accounts, or alternatively, close this and resubmit a new PR using only a single account.

**Comment by thiennguyen93** [comment]
Everything is good know

**Comment by thiennguyen93** [comment]
@letsbuildanything Please help me review this one

File: content/postgresql/concepts/comments/comments.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Tags:
+  - 'Comments'
+  - 'Documentation'
+CatalogContent:
+  - 'getting-started-off-platform-for-data-science'
+  - 'paths/design-databases-with-postgresql'
+  - 'paths/computer-science'
+---
+
+**Comments** are text placed inside code that is not executed. They are intended as documentation or explanation of the code they are a part of.
```
Comment (letsbuildanything) [inline content/postgresql/concepts/comments/comments.md:16]:
```suggestion
In PostgreSQL, **comments** are annotations added to SQL code to provide information about the code without affecting its execution. They are primarily used for documentation purposes and enhancing code readability. Comments in PostgreSQL can be single-line or multi-line and are initiated with -- for single-line comments and /* */ for multi-line comments.
```

File: content/postgresql/concepts/comments/comments.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Syntax
+
+In PostgreSQL, you can use either single-line or multi-line comments.
+
+- Single-line comments start with `--` and end at the end of the line.
```
Comment (letsbuildanything) [inline content/postgresql/concepts/comments/comments.md:22]:
```suggestion
Single-line comment:
```

File: content/postgresql/concepts/comments/comments.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+```sql
+-- This is a single-line comment
+```
+
+- Multi-line comments start with `/*` and end with `*/`.
```
Comment (letsbuildanything) [inline content/postgresql/concepts/comments/comments.md:28]:
```suggestion
Multi-line comment:
```

File: content/postgresql/concepts/comments/comments.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+```sql
+/*
+This is a
+multi-line comment
+*/
+```
+
+## Example
+
+Here's an example of how you can use comments in PostgreSQL:
```
Comment (letsbuildanything) [inline content/postgresql/concepts/comments/comments.md:39]: Avoid using first and second-person pronouns, i.e. "you", "I", and "we".

File: content/postgresql/concepts/comments/comments.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+SELECT *
+FROM employees; -- This comment is at the end of the line
+
+/*
+This is a multi-line comment
+that spans multiple lines
+*/
+SELECT *
+FROM departments;
+```
```
Comment (letsbuildanything) [inline content/postgresql/concepts/comments/comments.md:52]: Here, you should utilize single-line or multi-line comments to explain the given code. Remove these `single-line` or `multi-line` keyword comments and add the necessary comments.

File: content/postgresql/concepts/comments/comments.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+In the above example, comments are used to describe what the following SQL statement does. The PostgreSQL compiler will ignore these comments when executing the SQL code.
```
Comment (letsbuildanything) [inline content/postgresql/concepts/comments/comments.md:54]: Use a `Note` to inform the reader of points of potential confusion or to warn of potential pitfalls.
ex: `> **Note:** Note Text.`

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@thiennguyen93, Thank you for contributing to the Codecademy docs!
I've requested below few changes. Please make them as soon as possible.

**Comment by thiennguyen93** [comment]
@letsbuildanything Thank you for your code reviews. I have made some changes to fix all.
Thank again.

**Review by letsbuildanything — APPROVED** [review]
@thiennguyen93 
It seems ready for the next review.

**Review by mamtawardhani — APPROVED** [review]
@thiennguyen93 Thank you for contributing to Codecademy Docs!

The entry looks good to me.

**Pull request merged**